### PR TITLE
Sliders change live cell color

### PR DESCRIPTION
**Pull request by kevlarr**
- Adds RGB selector in sidebar to control live cell color
- Reorders the sidebar elements
- Cleans up unused/unnecessary classes and CSS styles

Screen Shot 2019-09-26 at 9 55 16 AM

### CODE REVIEW

File: lib/game_of_life_web/templates/universe/show.html.leex [hunk 1]
```diff
@@ -1,3 +1,9 @@
+<style>
+.cell.alive {
+  background: <%= render_color_rgb(@color) %>;
+}
+</style>
```
Comment (kevlarr) [inline lib/game_of_life_web/templates/universe/show.html.leex:6]: I was initially setting the `%Color{}` struct on the `%Universe{}` itself, and I was going to use that in `render_universe` to set the color inline on each cell, but it actually felt much cleaner to just add this `<style>` tag here and let the browser style engine do more of the work.

Comment (jprincipe) [reply lib/game_of_life_web/templates/universe/show.html.leex:6]: I really like this solution!

File: lib/game_of_life_web/views/universe_view.ex [hunk 2]
```diff
@@ -21,6 +22,10 @@ defmodule GameOfLifeWeb.UniverseView do
     end
   end
 
+  def render_color_rgb(%Color{} = color) do
```
Comment (jprincipe) [inline lib/game_of_life_web/views/universe_view.ex]: you could expand pattern matching here:

```elixir
def render_color_rgb(%Color{red: red, green: green, blue: blue}), do: "rgb(#{red},#{green},#{blue})"
```

Comment (jprincipe) [reply lib/game_of_life_web/views/universe_view.ex]: you could also add a function to the `Color` struct that generates this string

**Review by jprincipe — APPROVED** [review]
this is really well done

**Review by jprincipe — APPROVED** [review]

File: lib/game_of_life_web/views/universe_view.ex (hunk 2 above)
Comment (kevlarr) [reply lib/game_of_life_web/views/universe_view.ex]: Oooh I like that!

Comment (kevlarr) [reply lib/game_of_life_web/views/universe_view.ex]: Oh right, interpolation is a thing...

**Review by jprincipe — APPROVED** [review]

**Pull request merged**